rpcs3/Emu/Cell/lv2/lv2.cpp

```diff
@@ -49,11 +49,20 @@ static bool null_func(ppu_thread& ppu)
 	return false;
 }
 
+static bool uns_func(ppu_thread& ppu)
+{
+	LOG_TRACE(HLE, "Unused syscall %d -> ENOSYS", ppu.gpr[11]);
+	ppu.gpr[3] = CELL_ENOSYS;
+	ppu.cia += 4;
+	return false;
+}
+
 std::array<ppu_function_t, 1024> g_ppu_syscall_table{};
 
 // UNS = Unused
 // ROOT = Root
 // DBG = Debug
+// DEX..DECR = 	Unavailable on retail consoles
```

Comment:
Why you don't provide your tests? Or why you disabled sys_trace?

Reply:
@Nekotekina http://www.psdevwiki.com/ps3/LV2_Functions_and_Syscalls I've actually read it before opening the pr, sys_trace is debug only

Reply:
I've sent you the testcase on discord

Reply:
This wiki is very inexact source of information.